MaterialLibrary/src/main/java/io/doist/material/drawable/ColorMaterialDrawable.java

```diff
@@ -0,0 +1,77 @@
+package io.doist.material.drawable;
+
+import org.xmlpull.v1.XmlPullParser;
+import org.xmlpull.v1.XmlPullParserException;
+
+import android.content.Context;
+import android.content.res.Resources;
+import android.content.res.TypedArray;
+import android.graphics.drawable.ColorDrawable;
+import android.util.AttributeSet;
+
+import java.io.IOException;
+import java.lang.ref.WeakReference;
+
+import io.doist.material.R;
+import io.doist.material.reflection.ReflectionUtils;
+
+public class ColorMaterialDrawable extends ColorDrawable {
+    private final WeakReference<Context> mContext;
+
+    ColorMaterialDrawable(Context context) {
+        super();
+        mContext = new WeakReference<>(context);
+    }
+
+    @Override
+    public void inflate(Resources r, XmlPullParser parser, AttributeSet attrs)
+            throws XmlPullParserException, IOException {
+        // Don't call super, because it would crash on a theme attribute in android:color.
```

Comment:
We usually never call `super` in `Material<Drawables>`.
I'm not sure if we need that comment.